lib/exceptions/backends/log_result.rb

```diff
@@ -12,8 +12,9 @@ def initialize(backend, logger = ::Logger.new(STDOUT))
         @logger  = logger
       end
 
-      def notify(exception, options = {})
-        backend.notify(exception, options).tap do |result|
+      def notify(*args)
+        backend.notify(*args).tap do |result|
```

Comment:
nice. either way looks good to me, this is cleaner if it actually needs to do something with the args (which happens in some cases). i figure may as well not even mess with a `(*args)` if all it's doing is delegating. i will fix and update this PR